### DISCUSSION TITLE
Showcase your Open Source projects, contributions or profile

### DISCUSSION THREAD

**Discussion by julien-deramond** [Open-reSource/openresource.dev#3] [Show and tell]
Are you ready to showcase your Open Source projects, contributions, or profile to the world? You're in the right place! Join our community and let's connect. Don't be shy – show off your hard work and be proud of what you've accomplished. Let's celebrate the power of Open Source together!

All the links shared as comments here will be retrieved automatically by a script each week, and displayed randomly at https://www.openresource.dev/showcase.

**Comment by ArtyomVancyan** [comment]
- https://github.com/pysnippet/thumbnails
- https://github.com/pysnippet/django-forbid
- https://github.com/ArtyomVancyan/antd-phone-input

**Comment by fabiocaccamo** [comment]
- https://github.com/fabiocaccamo/FCUUID
- https://github.com/fabiocaccamo/django-admin-interface
- https://github.com/fabiocaccamo/python-benedict
- https://github.com/fabiocaccamo/FCFileManager
- https://github.com/fabiocaccamo/django-colorfield
- https://github.com/fabiocaccamo/django-treenode
- https://github.com/fabiocaccamo/django-maintenance-mode
- https://github.com/fabiocaccamo/django-extra-settings
- https://github.com/fabiocaccamo/FCCurrentLocationGeocoder
- https://github.com/fabiocaccamo/python-fsutil
- https://github.com/fabiocaccamo/FCIPAddressGeocoder
- https://github.com/fabiocaccamo/django-freeze
- https://github.com/fabiocaccamo/python-codicefiscale
- https://github.com/fabiocaccamo/django-redirects
- https://github.com/fabiocaccamo/python-fontbro
- https://github.com/fabiocaccamo/utils.js
- https://github.com/fabiocaccamo/create-matrix-action

**Reply by ArtyomVancyan** [reply]
Cool projects!

**Comment by patkon** [comment]
https://github.com/SwiftyEdit/SwiftyEdit

**Comment by diamant3** [comment]
- https://github.com/diamant3/psp-controller
- https://github.com/diamant3/SYSIG
- https://github.com/pspdev/pspsdk

**Comment by rochacbruno** [comment]
- https://github.com/dynaconf/dynaconf
- https://github.com/rochacbruno/flask-project-template
- https://github.com/rochacbruno/py2rs

**Comment by confused-Techie** [comment]
* https://github.com/pulsar-edit/pulsar

**Comment by ArtyomVancyan** [comment]
- https://github.com/pysnippet/fastapi-oauth2

**Comment by nkdas91** [comment]
- https://github.com/materialstyle/materialstyle
- https://github.com/marqueefy/marqueefy.github.io

**Comment by ajayk074** [comment]
- https://github.com/themeselection/vue-cheatsheet

**Comment by pranshugupta54** [comment]
- https://github.com/digitomize/digitomize
- https://github.com/digitomize/api-docs

**Comment by mevdschee** [comment]
https://github.com/mevdschee/php-crud-api

**Comment by nkdas91** [comment]
https://github.com/cuesync/cuesync.github.io

**Comment by joansebgarcia** [comment]
https://github.com/open-land/open-ui

**Comment by rossrobino** [comment]
https://github.com/rossrobino/drab

**Comment by trobonox** [comment]
https://github.com/trobonox/kanri

**Comment by gokulkrishh** [comment]
https://github.com/gokulkrishh/bmrk.cc

**Comment by crftwrk** [comment]
- https://github.com/bootscore/bootscore

**Comment by nousantx** [comment]
https://github.com/tenoxui/tenoxui

**Comment by castarco** [comment]
- https://github.com/KindSpells/astro-shield
- https://github.com/Coder-Spirit/nominal
- https://github.com/Coder-Spirit/beautiful-tree

**Comment by anav5704** [comment]
https://github.com/anav5704/CS000

**Comment by pylapp** [comment]
https://github.com/pylapp/Tips-tools
https://github.com/pylapp/tapsterbot
https://github.com/pylapp/SmoothClicker
https://github.com/Orange-OpenSource/floss-toolbox
https://github.com/Orange-OpenSource/Tota11ylost
https://github.com/Orange-OpenSource/La-Va11ydette
https://github.com/Orange-OpenSource/uuv
https://github.com/Orange-OpenSource/a11y-guidelines

**Comment by pylapp** [comment]
https://github.com/Orange-OpenSource/hurl

**Comment by azat-io** [comment]
https://github.com/azat-io/eslint-plugin-perfectionist
https://github.com/azat-io/todoctor
https://github.com/azat-io/eyecons

**Comment by retlim** [comment]
https://gitlab.com/valvoid/fusion/php/code

**Comment by sebousan** [comment]
https://github.com/Hugolify/hugolify-template
https://github.com/Hugolify/hugolify-theme
https://github.com/Hugolify/hugolify-admin

**Comment by Gothsec** [comment]
https://github.com/Gothsec/MoviesForDevs
https://github.com/Gothsec/Portfolio

**Comment by coliff** [comment]
https://github.com/htmlhint/htmlhint

**Comment by bobsingor** [comment]
https://github.com/embedpdf/embed-pdf-viewer

**Comment by onerrit** [comment]
https://github.com/tirrenotechnologies/tirreno

**Comment by azat-io** [comment]
https://github.com/azat-io/actions-up
https://github.com/azat-io/token-limit
https://github.com/azat-io/eslint-plugin-de-morgan

**Comment by zdunecki** [comment]
https://github.com/livesession/xyd

**Comment by FenjuFu** [comment]
https://github.com/iflytek/astron-agent
https://github.com/iflytek/astron-rpa
https://github.com/iflytek/skillhub
https://github.com/iflytek/astronclaw-tutorial

**Comment by hatemhosny** [comment]
https://github.com/live-codes/livecodes

**Comment by anviren** [comment]
https://github.com/synqratech/omega-walls

**Comment by shantanu-deshmukh** [comment]
https://github.com/shantanu-deshmukh/chunktuner

**Comment by blackmatriXblack** [comment]
https://github.com/blackmatriXblack/windowsmenu

**Comment by gregkonush** [comment]
https://github.com/proompteng/bilig

Bilig is a formula-backed WorkPaper runtime for Node services and agent tools. It lets agents and backend code edit cells through an API, recalculate formulas, verify exact readback, and persist JSON instead of driving a GUI spreadsheet.

The useful edge case is spreadsheet-style business logic in code: pricing rules, payout checks, import validation, quote approval, and agent-readable workbook state.

**Comment by blackmatriXblack** [comment]
https://github.com/blackmatriXblack/blackmatrixos



https://github.com/blackmatriXblack/physicswebos

https://github.com/blackmatriXblack/mathwebos

**Comment by 255kb** [comment]
https://github.com/mockoon/mockoon

**Comment by ilyario** [comment]
https://github.com/dataflow-operator/dataflow

**Comment by DevilsAutumn** [comment]
https://github.com/DevilsAutumn/quater

An opensource python backend framework for creating multi surface applications - means one handler but accessible via http, mcp or CLI.

**Comment by Simon-He95** [comment]
https://github.com/Simon-He95/markstream-vue

Markstream is a multi-framework streaming Markdown renderer for AI chat interfaces. It handles incomplete token streams with low-jitter updates and supports Vue/Nuxt, React/Next.js/Remix, Svelte, Angular, and Vue 2, including Mermaid, KaTeX, Shiki, Monaco, safe HTML, SSR, and custom components.

**Comment by Yohjisakamoto** [comment]
## Tura — local, open-source runtime harness

Tura is a local, open-source runtime harness for developers who are tired of vague skill claims, token-saving extensions with no evidence, and agents that change repositories without understanding them first.

- Source: https://github.com/Tura-AI/tura
- Website: https://turaai.net/
- Published benchmark, methodology, and artifacts: https://turaai.net/benchmark

In published DeepSWE v1.1 comparisons (20 tasks, 60 sessions, GPT-5.6 SOL High), Tura Direct used 83.5% fewer aggregate tokens than official Codex CLI High, with 65.0% vs 60.0% verifier success. Tura Balanced reached 80.0% success while using 49.6% fewer tokens. See the benchmark page for methodology, raw artifacts, and scope.

Disclosure: I’m affiliated with Tura-AI, and AI assistance was used to prepare this post; it was reviewed before publishing.

**Comment by FenjuFu** [comment]
Adding some detail to [my earlier comment](https://github.com/orgs/Open-reSource/discussions/3#discussioncomment-16340475), which was just bare links.

## iFlytek Astron — an open-source agent stack

Everything below is browsable under the [`iflytek-astron`](https://github.com/topics/iflytek-astron) topic: **9 repositories, all Apache-2.0, ~21.7k stars combined.**

| Repository | ★ | Lang | What it is |
|---|---|---|---|
| [iflytek/astron-agent](https://github.com/iflytek/astron-agent) | 9.5k | Java | Enterprise-grade agentic workflow platform — visual canvas, plugins, model management |
| [iflytek/astron-rpa](https://github.com/iflytek/astron-rpa) | 6.0k | Java | Agent-ready RPA suite with out-of-the-box automation tools |
| [iflytek/skillhub](https://github.com/iflytek/skillhub) | 4.8k | Java | Self-hosted agent **skill registry** for enterprises — publish, version, review, RBAC |
| [iflytek/astronclaw-tutorial](https://github.com/iflytek/astronclaw-tutorial) | 428 | JS | End-to-end tutorial for AstronClaw (cloud) and Loomy (desktop) |
| [harnessclaw/harnessclaw](https://github.com/harnessclaw/harnessclaw) | 341 | TS | Electron desktop app for managing AI agents |
| [harnessclaw/harnessclaw-engine](https://github.com/harnessclaw/harnessclaw-engine) | 281 | Go | LLM programming-assistant engine, WebSocket-based |
| [iflytek/iFly-Skills](https://github.com/iflytek/iFly-Skills) | 213 | Python | Skill collection: speech, OCR, translation, proofreading |
| [iflytek/memflywheel](https://github.com/iflytek/memflywheel) | 33 | TS | Agent-native long-term memory that learns after every run |
| [iflytek/domux](https://github.com/iflytek/domux) | 28 | Python | Low-latency model turning natural home commands into structured data |

`astron-agent` is listed in the **CNCF Landscape** under AI Agent → Workflow & Orchestration:

![Astron Agent in the CNCF Landscape](https://raw.githubusercontent.com/FenjuFu/astron-images/main/%E8%8B%B1%E6%96%87/CNCF_Landscape_AI_Agent_Workflow_Orchestration_Astron_Agent.png)

Astron and SkillHub are also listed as Associate Members of the **AAIF** (Agentic AI Foundation):

![Astron and SkillHub as AAIF Associate Members](https://raw.githubusercontent.com/FenjuFu/astron-images/main/%E4%B8%AD%E6%96%87/AAIF%E5%AE%98%E7%BD%91_Associate_Members_Astron_SkillHub.png)

**SkillHub** — the piece I find most interesting: a self-hosted registry for agent skills, so a team can publish, review and version `SKILL.md` packages internally instead of copy-pasting prompts around.

![SkillHub skill discovery](https://raw.githubusercontent.com/FenjuFu/astron-images/main/%E8%8B%B1%E6%96%87/skillhub_discover.png)

**HarnessClaw** — desktop client for running and orchestrating agents locally:

![HarnessClaw desktop](https://raw.githubusercontent.com/FenjuFu/astron-images/main/%E8%8B%B1%E6%96%87/HarnessClaw_desktop_home_connected.png)

Screenshots are hosted in "FenjuFu/astron-images"



---

*Disclosure: I'm a community member of the iFlytek Astron ecosystem and a contributor to several of these repositories. They are developed by iFlytek and partner teams, not by me personally.*

**Comment by lonestill** [comment]
## Onyx Launcher

An open-source Minecraft launcher for Windows and Linux with isolated instances, Modrinth integration, automatic Java management, Crash Bisect, performance diagnostics, World Guard snapshots, and safe backups.

- Source: https://github.com/lonestill/onyx-launcher

**Comment by bobleer** [comment]
## BitFun

BitFun is an open-source desktop agent built around a Rust runtime. The odd bit: a task can turn into a stateful Mini App — a form, board, chart, or tool — instead of ending as another chat transcript. It also works in repos, terminals, and the browser, and the cross-device relay can be self-hosted.

- Source: https://github.com/GCWing/BitFun

Disclosure: I help maintain it.

**Comment by cpys** [comment]
## Codex Quota Overlay — quota and reset status beside Codex Desktop

Codex Quota Overlay is an MIT-licensed desktop companion for Codex Desktop on Windows and macOS. It shows remaining quota, the next reset time, and expiring reset credits beside the active conversation title, then hides whenever Codex is not the foreground app.

Privacy and implementation: it reads the documented local `account/rateLimits/read` method; it does not capture screenshots, read conversation titles, browser cookies, or account tokens; and the overlay is click-through and never takes focus. Windows 11 is verified on physical hardware. macOS packages have automated build checks, while physical-Mac acceptance is still pending and labeled accordingly.

Source and downloads: https://github.com/cpys/codex-quota-overlay

Disclosure: I maintain the project. It is an independent community project and is not affiliated with or endorsed by OpenAI. Feedback on placement, compatibility, or the short diagnostic codes is welcome.

**Comment by Ducksss** [comment]
## Payload Components — wired Payload CMS blocks as owned source

[Payload Components](https://github.com/Ducksss/payload-components) is an MIT, community-first registry and TypeScript CLI with 67 typed blocks for Payload CMS v3 + Next.js.

Each block installs as source code the consuming project owns. The CLI also handles the CMS-specific integration work: it registers the block in the Pages collection, maps it in RenderBlocks, regenerates Payload types and the admin import map, and records idempotent install state. The result is a normal, reviewable git diff rather than an opaque runtime dependency.

Maintainers can inspect every planned file copy and wiring edit before changing a project:

```bash
npx payload-components add hero-basic --dry-run
```

- Source: https://github.com/Ducksss/payload-components

The project is actively maintained and welcomes block proposals, bug reports, documentation improvements, and pull requests.

Disclosure: I maintain Payload Components.

**Comment by wsdt** [comment]
## SEMAPRAX — semantic tooling for agent edits

SEMAPRAX is an Apache-2.0, experimental pre-alpha language and compiler research project implemented in Rust. It explores persistent declaration identities, checked HIR, deterministic semantic graph projections, revision-bound patches, and independent evidence replay before mutation. Human-readable `.spx` source remains the canonical Git representation.

- Source: https://github.com/wavect/semaprax

The project does not claim production readiness, complete memory safety, or completion of its broader systems-language goal; current implementation status is tracked through an evidence-gated completion matrix.

Disclosure: I am submitting this on behalf of Wavect GmbH, which develops SEMAPRAX. OpenAI Codex assisted with preparing this description; Wavect reviewed the claims.

**Reply by anothonybrown11** [reply]
This is a really creative way to explore open-source projects. I like the visual approach

**Comment by Mr-ZhangBo** [comment]
## AI Model Prompt Brief Builder — model-aware briefs for AI image and video workflows

AI Model Prompt Brief Builder is an MIT-licensed, dependency-free browser tool that turns creative requirements into structured, reusable briefs for AI image and video generation.

Instead of treating every model as the same prompt box, it separates model-independent intent—subject locks, composition, reference roles, camera and motion, negative constraints, delivery format, and acceptance criteria—from provider-specific controls. The current model notes cover GPT Image, FLUX, Seedream, Nano Banana, Veo, Seedance, MiniMax, and Grok Imagine workflows.

The tool runs as static HTML, CSS, and JavaScript with no build step. It is useful for prompt engineering, multi-model routers, marketing asset workflows, and repeatable model comparisons.

- **Source:** https://github.com/Mr-ZhangBo/ai-model-prompt-brief-builder

**Disclosure:** I maintain this project. Feedback on the schema, model adapters, and validation fields is welcome.

**Comment by iamshizzy6-creator** [comment]
Hi everyone! 👋

I’d like to showcase Repolis, an experiment that explores a different way to discover open-source projects.

Repolis turns public GitHub repository data into an interactive virtual town, where repositories can be explored in a more visual way.


GitHub repository:
https://github.com/hyeonsangjeon/Repolis

I’d love to hear feedback from the open-source community. Thanks!

**Reply by anothonybrown11** [reply]
This is a really creative way to explore open-source projects. I like the visual approach

**Reply by benjaminharris46788-prog** [reply]
Nice work

**Reply by shizzydigitalweb-hash** [reply]
Would like to know more about these project

**Reply by adekunleabdulkabir85-blip** [reply]
Would be cool to see how this scales with thousands of repositories

**Reply by jaspixelg** [reply]
I have engage on your repo
Will be happy if you can do the same

**Reply by iamshizzy6-creator** [reply]
> Would like to know more about these project



> This is a really creative way to explore open-source projects. I like the visual approach

Thank you! I’m glad you like the visual approach. The goal is to make discovering open-source projects feel more engaging and easier to explore.

**Reply by iamshizzy6-creator** [reply]
> Nice work

Thank you! I really appreciate the feedback.

**Reply by iamshizzy6-creator** [reply]
> Nice work

Thank you! I really appreciate the feedback.

**Reply by iamshizzy6-creator** [reply]
> Would be cool to see how this scales with thousands of repositories

Good question. Repolis already uses level-of-detail rendering for
buildings, batches repeated scenery with instancing, and reduces
decorative work on lower-end devices. Those help with rendering cost,
but they are not a guarantee of smooth performance with thousands of
repositories.

The quick public-profile view (?user=...) currently fetches up to 100
public repositories and marks coverage as partial when that limit is
reached. The owner's town uses a generated JSON catalog refreshed
daily; it does not query GitHub for every building during rendering.

So the current experience should not be presented as a proven
thousand-repository city. At that scale, we would need dedicated
loading, memory, and frame-time measurements. Loading districts in
chunks is one possible direction to explore, rather than constructing
everything at once, but it is not a feature we are claiming is
implemented today."

The current source and limitations are here:
https://github.com/hyeonsangjeon/Repolis/blob/main/index.html
https://github.com/hyeonsangjeon/Repolis/blob/main/docs/known-limitations.md

**Comment by sridharkalaibala** [comment]
## Posnic POS

Posnic is an AGPL-3.0 open source POS and billing software project for retail shops and restaurants. It is offline-first, with desktop releases for local operation and a self-hosted server option for online/offline POS workflows.

- Source and releases: https://github.com/Posnic/POS

Disclosure: I am submitting this on behalf of the Posnic project.

**Comment by adekunleabdulkabir85-blip** [comment]
Hi, thanks for reaching out and sharing your project. I took a look at the
POS repository, and I can see there’s a good opportunity to increase its
visibility and get it in front of more relevant developers and potential
users.
My approach would focus on targeted open-source promotion, rather than
artificial numbers. I’d promote it through relevant GitHub communities,
Reddit, Discord, developer forums, X, and other places where people
interested in POS systems, retail software, restaurant technology, billing
solutions, and offline-first applications are active.
The goal would be to attract real users, developers, contributors, stars,
forks, and discussions around the project.
I offer Basic, Standard, and Premium promotion plans, depending on how
aggressively you want to grow the project.

**Comment by eilinwis** [comment]
Hey everyone! 👋

I'm so happy to present you my chat web application where you can learn e2e testing with Playwright (and even coding with React)
11 lessons course with real code review from myself.

GitHub repository:
https://github.com/eilinwis/playwright-chat-lab

I would be happy for any kind of feedback!. Thanks!

**Comment by InsightFactoryAPP** [comment]
Hi everyone 👋

Sharing YYLO — an open-source CLI for orchestrating multiple AI coding agents (Codex, Claude Code, Gemini CLI and more) from one terminal: shared Kanban/task state, git worktree isolation per task, and typed merge flows.

https://github.com/yylo-dev/yylo

MIT-licensed, npm-installable, actively developed daily. Happy to join the showcase!

**Comment by mevdschee** [comment]
Want to play minesweeper on Mac? Now you can!

https://github.com/mevdschee/fyne-mines
